### PR TITLE
Keypair from seed

### PR DESCRIPTION
**Pull request by ailisp**
Borrow from rust code, and verified by using the `near` seed account to create accounts in devnet

### CODE REVIEW

File: src.ts/utils/key_pair.ts [hunk 1]
```diff
@@ -86,6 +86,20 @@ export abstract class KeyPair {
             throw new Error('Invalid encoded key format, must be <curve>:<encoded key>');
         }
     }
+    
+    static fromSeed(seedStr: string): KeyPair {
```
Comment (vgrichina) [inline src.ts/utils/key_pair.ts:90]: What is the purpose for this, do we need to support this specific format of seed phrases?

Note that we already have library with support for conforming BIP39 seed phrases (compatible with other wallets):
https://github.com/nearprotocol/near-seed-phrase/blob/master/index.js

File: src.ts/utils/key_pair.ts [hunk 2]
```diff
@@ -86,6 +86,20 @@ export abstract class KeyPair {
...
+        const seed = new Uint8Array(32);
+        for (let i = 0; i < 32; i++) {
```
Comment (vgrichina) [inline src.ts/utils/key_pair.ts:92]: it seems like using only 32 characters from a phrase which supposedly contains common words doesn't provide nearly as much entropy as needed for strong 256-bit key. It's basically only enough for about 6 word seed phrase. I suggest we abandon this scheme altogether (i.e. not use it in Rust either).

Comment (bowenwang1996) [reply src.ts/utils/key_pair.ts:92]: I think we use this in rust mostly for testing purposes.

File: src.ts/utils/key_pair.ts (hunk 1 above)
Comment (ailisp) [reply src.ts/utils/key_pair.ts:90]: Good point! seems this change isn't very helpful for a general api-js user and they should seek for near-seed-phrase

**Pull request closed without merging**